releasing/README.md

```diff
@@ -4,44 +4,202 @@
 [`cloud-build-local`]: https://github.com/GoogleCloudPlatform/cloud-build-local
 [Google Cloud Build]: https://cloud.google.com/cloud-build
 [semver]: https://semver.org
+[Go modules]: https://github.com/golang/go/wiki/Modules
 
-Scripts and configuration files for publishing a
-`kustomize` release on the [release page].
+This document describes how to perform a [semver] release
+of one of the [Go modules] in this repository.
 
-## Build a release locally
+These modules release independently.
 
-Install [`cloud-build-local`], then run
+| Module Description | Module Prefix | Tag | Branch Name | 
+| ---         | ----        | --- | --- |
+| the kustomize executable | `sigs.k8s.io/kustomize/kustomize` | _kustomize/v{major}.{minor}.{patch}_ |_release-kustomize-v{major}.{minor}_ |
+| the kustomize Go API | `sigs.k8s.io/kustomize`   | _v{major}.{minor}.{patch}_ |_release-api-v{major}.{minor}_ |
+| the pluginator executable | `sigs.k8s.io/kustomize/pluginator` | _pluginator/v{major}.{minor}.{patch}_ | _release-pluginator-v{major}.{minor}_ |
+
+
+## Module summaries
+
+### sigs.k8s.io/kustomize/kustomize
+
+The `kustomize` program, a CLI for performing
+k8s-aware structured edits to YAML in k8s resource
+files.
+
+#### Packages
+
+There's only one package in this module.  It's called `main`,
+and it holds the _kustomize_ executable.
+
+
+#### Release artifacts
+
+Executable files appear for various operating systems
```

Comment:
Add the name of release artifacts like kustomize/v3.2.1

Reply:
done